docs/src/guide/installation.md

```diff
@@ -339,3 +413,4 @@ avrdude --version
 [wingit]: https://git-scm.com/download/win
 [winterm]: https://github.com/Microsoft/Terminal
 [winwsl]: https://docs.microsoft.com/en-us/windows/wsl/about
+    
```

Comment:
Nope.

Reply:
Under Linux a text file is [required to have a newline at the end](https://stackoverflow.com/a/729795). Some tools may fail otherwise